### ISSUE TITLE
Uncaught TypeError: Cannot read property 'onDidReloadKeymap' of undefined

### ISSUE THREAD

**Issue by torifat** [atom/settings-view#463]
**Atom Version**: 0.190.0
**System**: Mac OS X 10.10.2
**Thrown From**: [settings-view](https://github.com/atom/settings-view) package, v0.187.0

### Stack Trace

Uncaught TypeError: Cannot read property 'onDidReloadKeymap' of undefined

```
At /Applications/Atom.app/Contents/Resources/app/node_modules/settings-view/lib/keybindings-panel.js:119

TypeError: Cannot read property 'onDidReloadKeymap' of undefined
  at KeybindingsPanel.module.exports.KeybindingsPanel.initialize (/Applications/Atom.app/Contents/Resources/app/node_modules/settings-view/lib/keybindings-panel.js:119:39)
  at KeybindingsPanel.View (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:184:25)
  at new KeybindingsPanel (/Applications/Atom.app/Contents/Resources/app/node_modules/settings-view/lib/keybindings-panel.js:18:53)
  at /Applications/Atom.app/Contents/Resources/app/node_modules/settings-view/lib/settings-view.js:143:16
  at SettingsView.module.exports.SettingsView.getOrCreatePanel (/Applications/Atom.app/Contents/Resources/app/node_modules/settings-view/lib/settings-view.js:287:19)
  at SettingsView.module.exports.SettingsView.showPanel (/Applications/Atom.app/Contents/Resources/app/node_modules/settings-view/lib/settings-view.js:323:24)
  at HTMLAnchorElement.<anonymous> (/Applications/Atom.app/Contents/Resources/app/node_modules/settings-view/lib/settings-view.js:131:24)
  at space-pen-div.jQuery.event.dispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/jquery/dist/jquery.js:4430:9)
  at space-pen-div.elemData.handle (/Applications/Atom.app/Contents/Resources/app/node_modules/jquery/dist/jquery.js:4116:28)

```

### Commands

```
  4x -0:27.7 application:new-file (ul.list-inline.tab-bar.inset-panel)
     -0:16.6 tree-view:toggle (atom-text-editor.editor)
     -0:03.5 settings-view:open (ul.list-inline.tab-bar.inset-panel)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "file-icons",
      "css-color-highlighting",
      "autocomplete-html",
      "autocomplete-css",
      "autocomplete-plus",
      "travis-ci-status",
      "laravel",
      "language-jade",
      "language-ini",
      "angularjs-styleguide-snippets",
      "pain-split",
      "tabs-to-spaces",
      "atom-jasmine",
      "hex",
      "language-js-next"
    ],
    "themes": [
      "one-dark-ui",
      "one-dark-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
Stylus, v0.7.0
atom-alignment, v0.12.0
editorconfig, v0.3.3
emmet, v2.3.7
git-tab-status, v1.7.0
linter, v0.12.0
react, v0.11.2

# Dev
No dev packages
```

**Comment by kirollosmagdy** [comment]
hello, can you please look into that!

**Atom Version**: 0.190.0
**System**: Mac OS X 10.10.2
**Thrown From**: [settings-view](https://github.com/atom/settings-view) package, v0.187.0

### Stack Trace

Uncaught TypeError: Cannot read property 'onDidReloadKeymap' of undefined

```
At /Users/kirollosmagdy/Downloads/Atom.app/Contents/Resources/app/node_modules/settings-view/lib/keybindings-panel.js:119

TypeError: Cannot read property 'onDidReloadKeymap' of undefined
  at KeybindingsPanel.module.exports.KeybindingsPanel.initialize (/Users/kirollosmagdy/Downloads/Atom.app/Contents/Resources/app/node_modules/settings-view/lib/keybindings-panel.js:119:39)
  at KeybindingsPanel.View (/Users/kirollosmagdy/Downloads/Atom.app/Contents/Resources/app/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:184:25)
  at new KeybindingsPanel (/Users/kirollosmagdy/Downloads/Atom.app/Contents/Resources/app/node_modules/settings-view/lib/keybindings-panel.js:18:53)
  at /Users/kirollosmagdy/Downloads/Atom.app/Contents/Resources/app/node_modules/settings-view/lib/settings-view.js:143:16
  at SettingsView.module.exports.SettingsView.getOrCreatePanel (/Users/kirollosmagdy/Downloads/Atom.app/Contents/Resources/app/node_modules/settings-view/lib/settings-view.js:287:19)
  at SettingsView.module.exports.SettingsView.showPanel (/Users/kirollosmagdy/Downloads/Atom.app/Contents/Resources/app/node_modules/settings-view/lib/settings-view.js:323:24)
  at HTMLAnchorElement.<anonymous> (/Users/kirollosmagdy/Downloads/Atom.app/Contents/Resources/app/node_modules/settings-view/lib/settings-view.js:131:24)
  at space-pen-div.jQuery.event.dispatch (/Users/kirollosmagdy/Downloads/Atom.app/Contents/Resources/app/node_modules/jquery/dist/jquery.js:4430:9)
  at space-pen-div.elemData.handle (/Users/kirollosmagdy/Downloads/Atom.app/Contents/Resources/app/node_modules/jquery/dist/jquery.js:4116:28)

```

### Commands

```
  6x -0:54.4 core:move-down (atom-text-editor.editor.mini)
  3x -0:52.9 core:move-up (atom-text-editor.editor.mini)
  2x -0:52.2 core:backspace (atom-text-editor.editor.mini)
  7x -0:44.9 core:move-down (atom-text-editor.editor.mini)
  2x -0:43.4 core:move-up (atom-text-editor.editor.mini)
  4x -0:43.0 core:move-down (atom-text-editor.editor.mini)
  3x -0:42.3 core:move-up (atom-text-editor.editor.mini)
  3x -0:40.7 core:move-down (atom-text-editor.editor.mini)
     -0:33.8 core:move-up (atom-text-editor.editor.mini)
  4x -0:33.2 core:move-down (atom-text-editor.editor.mini)
  6x -0:32.4 core:move-up (atom-text-editor.editor.mini)
     -0:30.6 editor:consolidate-selections (atom-text-editor.editor.mini)
     -0:30.6 core:cancel (atom-text-editor.editor.mini)
  8x -0:20.4 window:decrease-font-size (atom-text-editor.editor)
     -0:16.6 window:increase-font-size (atom-text-editor.editor)
     -0:10.8 settings-view:open (atom-text-editor.editor)
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
No installed packages

# Dev
No dev packages
```

**Comment by mnquintana** [comment]
This has been fixed for the next release of Atom (0.191.0) in c1d68bc9fdfc111a2d0941d6cc9b44ce3ff8c8ff
